source/_patterns/20-molecules/teaser/teaser.scss

```diff
@@ -467,6 +459,11 @@
   .teaser__opinion {
     flex: 0 0 auto;
 
+    @include desktop-up {
```

Comment:
Mötzle auf hächstem Niveau: desktop-up nach tablet-up reintun, quasi sortiert nach Browser-Breite.

Reply:
dat is von dir ;-)